docs/getting-started/join-testnet.md

```diff
@@ -98,6 +98,37 @@ Note we use the `latest` directory in the [testnets repo](https://github.com/cos
 which contains details for the latest testnet. If you are connecting to a different testnet, ensure you
 get the right files.
 
+You now need to generate a `genesis tx` to generate a signed self-delegation:
```

Comment:
- This would belong in the `validator-setup.md` file
- Apparently we don't need it for GoS or gaia-9000.

Reply:
Updated